config/src/config/test_data/public_full_node.config.yaml

```diff
@@ -0,0 +1,18 @@
+base:
+    data_dir: "/opt/libra/data/common"
+    role: "full_node"
+    waypoint:
+        type: "from_config"
```

Comment:
[For later]: This `type` param is ugly for the user, ideally the type would be inferred from the presence of a `waypoint` or `backend` section.

Reply:
I'll see what I can do with these, I suspect there may be some limitations due to the use of serde / dynamic typing.